admin/tools/docker/deployment/kubernetes/env.example.sh

```diff
@@ -5,6 +5,7 @@
 # VERSION can be a git ticket branch but with _ instead of /
 # example: tickets_DM-7139, or dev
 VERSION=dev
+MARIADB_VERSION=10.1.25
```

Comment:
What's the planned use for this variable, and why should we depend in these scripts on a very specific version of the database server? I think this info could be obtained from the LSST Stack.

Reply:
The final goal is to remove dependency between qserv and mariadb in eups, so that qserv build do no more require mysql build and install, but only mysql client library. It will produce a lighter container for Qserv.
That's why I now run mariadb in a separate container, I currently use official mariadb container, but we could decide to use a eups build of mariadb in the middle term to unify version management. You're perfectly right, we have to discuss that.